### DISCUSSION TITLE
Dark Matter: A Baseless Hypothesis? | Pavel Kroupa, Francesca Chadha-Day, Bjørn Ekeberg

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#70] [Gravitation Models]
Dark-matter is falsified by observations - Discussion between Pavel Kroupa (pragmatic approach 👍 ), Francesca Chadha-Day (theoretical physicist), and Bjørn Ekeberg (philosopher and ACG member 👍 ), and how Modified Newtonian Gravity (MOND) accounts for these observations.

The first 16 minutes of the discussion is on YouTube's *The Institute of Art and Ideas* [https://youtu.be/06Ngjb2CC2Y](https://youtu.be/06Ngjb2CC2Y)

**Comment by khuramonline** [comment]
Not only dark matter. MOND is also baseless. MOND is also "dark". It is "dark equation" that works only because it is made to work.

**Reply by HanDeBruijn** [reply]
Exactly! [__MOND = UAC__](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm) .

**Reply by RedshiftDrift** [reply]
"Dark" means that it does not interact with light.
The idea of a "dark equation" is a [category mistake](https://en.wikipedia.org/wiki/Category_mistake).

The MOND equation is a [mathematical term added to Newton's gravity (or inertia) equation](https://en.wikipedia.org/wiki/Modified_Newtonian_dynamics) to make gravitation fit observations.  That's the reason why it works for galaxies.  However, MOND proponents have known for some time that [it doesn't work for the dynamics of galaxy clusters](https://www.aanda.org/articles/aa/full_html/2017/02/aa29358-16/aa29358-16.html), and they are also discovering that [MOND doesn't work with wide binaries](https://tritonstation.com/2023/05/18/wide-binary-weirdness/).  So the MOND hypothesis is failing.

Now, since the *MOND hypothesis* is failing and *MOND = UAC*, doesn't that mean that UAC also fails to explain the dynamics of galaxy clusters and wide binaries?

**Reply by ftessier** [reply]
MOND brings to mind the book [Reinventing gravity](https://en.wikipedia.org/wiki/Reinventing_Gravity) by John Moffat, which I very much enjoyed. As an alternative to MOND, it conjectures a _fifth force,_ also resulting in a force at large distance that is greater than Newton's. I don't know enough to know if this idea fits or not with modern observations.

**Reply by sahil5d** [reply]
Milgrom's radial acceleration relation and McGaugh's
`v^2/r = GM/r^2 + GM/r^2 / (e^sqrt(GM/r^2 / (c^2/(2pi * 13.8Gly) - 1)`
have predictive power on hundreds of galaxies, so I think it's very wrong to discount it. It's not baseless. It literally is based on galactic rotation data, which it models parsimoniously.
https://twitter.com/sahil5d/status/1452740114761732097

Do you have something better than
[a star's centripetal acceleration around a galaxy] `= v^2/r = GM/r^2 + GM/r^2 / (e^sqrt(GM/r^2 / (c^2/(2pi * c/H)) - 1)` ?

**Reply by HanDeBruijn** [reply]
![both](https://github.com/a-cosmology-group/acg/assets/135433222/c2245ea3-0ca8-418f-bb39-23032aa2da44)

The _galaxy rotation curves_ can only be understood with the acceptance of __two__ time frames.
There is an Orbital time frame for Gravity. And there is an Optical/Atomic time frame for EM/QM.
At an Earthly scale the two time frames are only different by a few milliseconds in a year.
But at the scale of a galaxy, the differences are far more significant.

Gravity does not work with an atomic time scale. Optics does not work with an orbital time scale.
That's why all Gravity models exhibit sort of a beginning, the worst case of which is a Big Bang.
And that's why all Plasma / Optical models are embedded in a Static Euclidean Universe (SEU).

[MOND = UAC](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm) is the __unification__ of Orbital time for Gravity and Optical time for Observation:

$$
\mbox{Observation} \sim \sqrt{\mbox{distance to nucleus}} \times \mbox{Calculation}
$$

UAC does not explain the dynamics of galaxy clusters and wide binaries simply because nobody has been looking at these subjects through the new (#63) glasses 😥

**Reply by RedshiftDrift** [reply]
@sahil5d 
> Milgrom's radial acceleration relation and McGaugh's
> $v^2/r = GM/r^2 + GM/r^2 / (exp(sqrt(GM/r^2 / (c^2/(2pi * 13.8Gly) - 1)))$
> have predictive power on hundreds of galaxies, so I think it's very wrong to discount it. 

It doesn't work exactly like that - MOND works on hundreds of galaxies, but it was designed for that.
MOND as a phenomenological model for galaxy rotation works fine, but MOND is actually a model of gravity.  Its **predictive power** would be better demonstrated with different situations, e.g. for big and small galaxies (where it works fine), for galaxy clusters (where it's off by a factor of three), for relativistic cases (there are now three(?) relativistic MOND extensions, and none of them works well), and *wide binaries* (where researchers don't agree yet.)

On the topic of wide binaries, Stacy McGaugh posted a short review of the conference [`MOND at 40` on his blog](https://tritonstation.com/2023/06/24/the-mond-at-40-conference/)  It's not clear what's happening but the researchers have good ideas for further tests.

**Reply by sahil5d** [reply]
Sure, there isn't yet a perfect understanding of galaxy clusters and wide binaries.

Also, I recall that the galaxy cluster issue was a mass estimate off by a factor of 2 not 3.
https://twitter.com/sahil5d/status/1564358858461544448

Whatever the ultimate understanding of gravity is, the way in which [a star's centripetal acceleration around a galaxy] will be modeled will exactlybe xor resemble Milgrom's and McGaugh's radial acceleration relations. If the very minimal modification to GM/r^2 of one constant `c^2/87Gly` and one functor `sqrt()` works, then the ultimate understanding can't be far off.

**Comment by khuramonline** [comment]
I do have called MOND to be "dark equation" in my book [Philosophy Unscrambles Dark matter](https://zenodo.org/record/3557000#.XsbOWUQzbIU). I have also explained the reasons for this treatment.

![image](https://github.com/a-cosmology-group/acg/assets/132253587/18d5ac44-a443-40f1-97cb-92090592f6f9)
![image](https://github.com/a-cosmology-group/acg/assets/132253587/a99789b0-be5c-4bb1-ac8a-dafbcb60caa5)
![image](https://github.com/a-cosmology-group/acg/assets/132253587/21f29944-704e-4f78-bec5-467f96e0ba02)

With this book since I know that:

1. Galaxies Cluster dynamics are not the proof of dark matter,
2. Galactic rotation curves are not the proof of dark matter,
3. Gravitational lensing is not the proof of dark matter,
4. Bullet Cluster is not the proof of dark matter,
5. Dark matter ring is not the proof of dark matter,
6. also there is no proof of dark matter in CMB or structure formation.

**So there is no dark matter at all. There is no need of MOND when there is no dark matter in all the above things.**

**Comment by khuramonline** [comment]
MOND works for -- or has been successfully made to work for galactic rotations. This fact indicates following possibilities:

1. Dark matter is not any type of matter but is a principle of nature that has been captured rightfully or only by phenomenological way by MOND.
2. Dark Matter is a systematic error in the official theory.
3. There is no error in the official theory. However, dark matter is a systematic error in the application of theory.

My book (2019) has explained that actually it is the case No.3.

Theory was already capable to explain galactic rotation curves. But scientists did commit systematic error in the application of right theory. They applied irrelevant Newton's Theorem instead of applying right relevant Newton's Theorem.

Since they applied irrelevant Newton's Theorem, so naturally they were trapped in a bogus but systematic problem.

Majority of scientists resolved the bogus systematic problem by introducing a ghost solution i.e. dark matter.

A small group of scientists resolved the bogus systematic problem by introducing another bogus but counter systematic adjustments of the theory.

Both approaches were simply wrong.

**Comment by khuramonline** [comment]
And "wide binaries"

They should not be conclusive. Even if they find that binary systems rotate by way of flat rotation curves ... that will not be helpful. Because the normal mass of the rest of the galaxy would act like theoretical dark matter. And those flat rotations would actually be caused by the normal matter of the surrounding galaxy.
